web-server/src/components/Teams/CreateTeams.tsx

```diff
@@ -191,15 +193,21 @@ const TeamRepos: FC = () => {
           getOptionLabel={(option) => `${option.parent}/${option.name}`}
           renderInput={(params) => (
             <TextField
+              onFocus={searchFocus.true}
+              onBlur={searchFocus.false}
               onChange={(e) => {
                 handleReposSearch(e as React.ChangeEvent<HTMLInputElement>);
                 searchQuery.set(e.target.value);
               }}
               {...params}
               label={
-                selectedRepos.length
-                  ? `${selectedRepos.length} selected`
-                  : `Search repositories`
+                selectedRepos.length ? (
+                  `${selectedRepos.length} selected`
+                ) : !searchFocus.value ? (
+                  <AnimatedInputWrapper />
+                ) : (
+                  'org/repo'
```

Comment:
<img width="278" alt="image" src="https://github.com/middlewarehq/middleware/assets/76566992/0d41aa40-8377-4033-a4e0-599dda8cef15">

it should have said `Search for org/repo`
https://github.com/middlewarehq/middleware/pull/448#discussion_r1660752967

Reply:
Done